auth-web/src/views/auth/HomeView.vue

```diff
@@ -83,22 +83,30 @@
       <div class="contact-info-container">
         <v-container>
           <v-row>
-            <v-col cols="12" md="8">
+            <v-col cols="12" md="7">
               <h3 class="mb-6">Need more information?</h3>
-              <p class="mb-4">To learn more about Cooperative Associations in British Columbia, please visit the Cooperative Associations information page on the <a href="https://www2.gov.bc.ca/gov/content/employment-business/business/managing-a-business/permits-licences/businesses-incorporated-companies/cooperative-associations" target="_blank" rel="noopener">BC Government website</a>.</p>
+              <p class="mb-4">To learn more about Cooperative Associations in British Columbia, please
+                <a :href="coopAssocUrl" target="_blank" rel="noopener noreferrer">
+                  visit the Cooperative Associations information page
+                </a>.
+              </p>
               <a class="link-w-icon" href="https://www2.gov.bc.ca/gov/content/employment-business/business/managing-a-business/permits-licences/news-updates/modernization/coops-services-card"
                  target="_blank" rel="noopener noreferrer">
-                <v-icon small class="mr-2">mdi-open-in-new</v-icon>
-                <span>Frequently Asked Questions</span>
+                <span>Cooperatives Online Frequently Asked Questions</span>
```

Comment:
Should this be "BC Business Registry" instead?

Reply:
Not sure of their future plans, but the designs as is include these.. 

<img width="556" alt="Screen Shot 2020-06-25 at 3 44 05 PM" src="https://user-images.githubusercontent.com/53542131/85802911-c6b43700-b6fa-11ea-90b2-ec6507f5fd9f.png">

Reply:
The content looks fine but I thought we got away from calling this Cooperatives Online...?

Reply:
Good point, i'll follow up on this.

Reply:
It seems it will stay this way until the BCOMPs FAQ is complete or possibly when we migrate everything away from this title. On the BCGov site, a lot of things are still titled Cooperatives Online. 

We're agile :) We can update at anytime!